sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -773,26 +773,7 @@ private[client] class Shim_v0_13 extends Shim_v0_12 {
     filters.flatMap(convert).mkString(" and ")
   }
 
-  private def quoteStringLiteral(str: String): String = {
```

Comment:
Why did you move this func?

Reply:
Didn't remove this method, just added the getPartitionsByFilterInternal method